error_tracker/django/utils.py

```diff
@@ -9,6 +9,7 @@
 import re
 import json
 from django.http import RawPostDataException
+from http.cookies import SimpleCookie
```

Comment:
No usage?

Reply:
oh ! yes. A bad copy/paste duplicate this on line 200. On next commit will be fix.

Reply:
just submit a fix, let me know.

Reply:
It seems http.cookies is not part of python 2.7 and it will fail there, that's the reason it's moved with try except

Reply:
ok, i would prefer this on headers

```
try:
    from http.cookies import SimpleCookie
except:
    pass
```

instead to see **from http.cookies import SimpleCookie** inside code.

ok for you ?

Reply:
But your code would down the line, When you're trying to use SimpleCookie module, you need to add some flag to say this is not available.